core/src/main/cpp/gdn/projection_mesh/projection_mesh.cpp

```diff
@@ -255,5 +255,13 @@ void ProjectionMesh::update_shader_param(int index, const String &param,
     }
 }
 
+void ProjectionMesh::set_camera_position(Vector3 camera_position) const {
+    update_shaders_param(kCameraPositionParamName, camera_position);
+}
+
+void ProjectionMesh::set_camera_xaxis(Vector3 camera_xaxis) const {
+    update_shaders_param(kCameraXaxisParamName, camera_xaxis);
+}
+
```

Comment:
This should be merged into a single method to make sure the caller does not forget to update the other when updating one.

Reply:
Done.